spec/models/ingredient_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Ingredient, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  it { should validate_presence_of(:food) }
+  it { should validate_presence_of(:recipe) }
+  it { should validate_presence_of(:amount) }
+
+  it { should belong_to(:food) }
+  it { should belong_to(:recipe) }
+
+  describe '#amount_per_serving' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.